components/cam/bld/namelist_files/namelist_definition.xml

```diff
@@ -2054,6 +2054,12 @@ Specify if cloud fraction is applied to microphysics tendencies.
 Default: TRUE
 </entry>
 
+<entry id="do_prescribed_CCN" type="logical" category="microphys"
+       group="micro_nl" valid_values="" >
+Specify if CCN concentration is read in from ncdf file within p3.
+Default: FALSE
```

Comment:
Here you say the default is "False" but it is actually true in `namelist_defaults_cam.xml`

Reply:
I just commented on this above! I say we leave this version as-is and change the previous mention to false.

Reply:
Fixed. Default is false now.